pants_jupyter_plugin/plugin.py

```diff
@@ -11,79 +11,22 @@
 import sys
 from contextlib import contextmanager
 from dataclasses import dataclass
-from typing import Any, Awaitable, Callable, Container, Iterable, Iterator, Optional, Tuple
+from typing import Any, Awaitable, Callable, Iterable, Iterator, Optional, Tuple
 
 import ipywidgets
 import nest_asyncio
 from IPython.core.magic import Magics, line_magic, magics_class
 from IPython.display import Javascript, display
-from pex.pex_bootstrapper import bootstrap_pex_env
-from pex.variables import Variables
+from twitter.common.contextutil import environment_as, temporary_dir
 
 # TODO: replace or vendor these.
```

Comment:
comment should move to the `twitter.common` line.

Reply:
Hrm, autoformatter. I'll see what I can do.

Reply:
Fixed.